.gitlab-ci.yml

```diff
@@ -219,37 +225,37 @@ trigger_build_kernels:
 
 test_deb_x64:
   extends: .test_agent6
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:v$CI_PIPELINE_ID
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64${ECR_TEST_ONLY}:v$CI_PIPELINE_ID
```

Comment:
Tests are going to fail on `main` with that setup, as `ECR_TEST_ONLY` is not set to the empty string on `main` pipelines for this job (and the other test jobs).